spec/puppetserver/ca/cli_spec.rb

```diff
@@ -363,18 +378,9 @@
       expect(exit_code).to eq(1)
     end
 
-    it 'does not require the config option' do
-      Dir.mktmpdir do |tmpdir|
-        with_temp_cadir tmpdir do
-          exit_code = Puppetserver::Ca::Cli.run(['generate'], stdout, stderr)
-          expect(exit_code).to eq(0)
-        end
-      end
-    end
-
```

Comment:
I removed this since it was not writing to the tmpdir

Reply:
I think we may want to test this at a different level, like via calling the action's `parse` method, if at all.

Reply:
I'll add that @Magisus

Reply:
^ Actually, you moved/added more generate specs in your PR.  So maybe I should add it after you merge yours?